packages/kolibri/composables/useTour.js

```diff
@@ -0,0 +1,31 @@
+import { reactive, ref } from 'vue';
+
+const tourActive = ref(false);
+const tourActiveMap = reactive({});
+
+function startTour(pageName) {
+  setTimeout(() => {
```

Comment:
This isn't a huge deal, but I'd recommend also adding a comment above the `setTimeout` to clarify that the delay is intentional for UX reasons (letting users briefly see the page before the tour darkens it). Without that context, a future dev could assume it’s unnecessary and remove it, which would hurt the onboarding experience.